libbeat/docs/shared-docker.asciidoc

```diff
@@ -137,6 +137,24 @@ using this syntax:
 
 endif::apm-server[]
 
+==== Run {beatname_uc} on a read-only file system
+
+If you'd like to run {beatname_uc} in a Docker container on a read-only file 
+system, you can do so by specifying the `--read-only` option.
+{beatname_uc} requires a stateful directory to store application data, so
+with the `--read-only` option you also need to use the `--mount` option to
+specify a path to where that data can be stored.
+
+For example:
+
+["source", "sh", subs="attributes"]
+--------------------------------------------
+docker run \
+  --mount type=tmpfs,destination=/usr/share/{beatname_lc}/data
```

Comment:
@blakerouse Does the mount type necessarily need to be `tmpfs`?  I don't think so but please confirm.  If it doesn't, then I think we should remove it from here so as to not sound prescriptive about it?

Reply:
Correct it does not need to be tmpfs, and I don't recommend even using tmpfs. `tmpfs` should only be used in the case that they truly don't want to store any state (that can lead to data duplication though).

This would be better to have it like so:

```suggestion
  --mount type=source=$(pwd}/data,destination=/usr/share/{beatname_lc}/data
```

That would have the container write into the current directory sub-path `/data`.

Reply:
Thanks. Changed as suggested.